Add in item_equipment.sql for main hand or no MID items.

helixhamin opened the pull request:
Still need to find MIDs for all the other items missing before I start adding (Not currently included in this PR, for a future PR). Added some of the missing item mods, but I want to limit the amount of changes in the submission, so I will add more mods and latents to the next PR.

<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

<!-- Describe what your PR does here. If it closes an existing issue, you can mention: "Closes #1234" and GitHub will link this PR to that issue. -->

Adds missing items and some of their latents/mods/etc, to item_equipment and item_weapon

## Steps to test these changes

<!-- Clear and detailed steps to test your changes here -->

Update the SQL with the tables, use gm commands to additem, see if items added to equipment can be equipped, items added to weapon can we weaponed, and items added to mods can mod stats.

zach2good committed:
[core] Add bindings to get/set item exData

zach2good committed:
[chocobo] Add notes for egg exdata

helixhamin committed:
Add in item_equipment.sql for main hand or no MID items.
Still need to find MIDs for all the other items missing before I start adding.
Added some of the missing item mods, but I want to limit the amount of changes in the submission, so I will add more mods and latents to the next PR.

helixhamin commented:
Hmm, not sure why the "build / Full_Startup_Checks_Linux (pull_request)" failed. I only modified SQL, which shouldn't cause this, and didn't submit anything that changed it.

`hxi exited with code 255
Error: Child_process exited with error code 255`

`Error: 23 06:29:42:351][connect][error] Fatal: bind: Address already in use (ConnectServer:93)`

zach2good committed:
[dynamis] Refactor currency NPCs

zach2good committed:
[dynamis] Add ENABLE_EXCHANGE_100S_TO_1S setting

Co-Authored-By: CynicaGaming <47292146+CynicaGaming@users.noreply.github.com>

zach2good committed:
[dynamis] Rename hundo -> hundred

zach2good committed:
[quest] Add missing TOAU quest enums

Co-Authored-By: InoUno <inoiown@gmail.com>
Co-Authored-By: KnowOne134 <35616771+KnowOne134@users.noreply.github.com>
Co-Authored-By: Jasson McMorris <jasson@siegegames.com>

Xaver-DaRed committed:
Fix Windurst Airship aimation

loxleygames committed:
Get item ID by name or sortname

Skerxan committed:
[Shops] Audited Royal Armoury

Audited the 2 shop NPCs in Northern Sandy. Arlenne & Tavourine

- Fixed prices
- Fixed nation placement requirements
- Fixed order of items
- Added missing weapons (Bronze Spear, Brass Spear, Spear)

I compared BGwiki to JPwiki and noticed that JPwiki has the updated list of the July 2011 patch. (They even list the changes on their shop page.)
Also the JP list has a note that their prices are the base prices untouched by fame etc. (The numbers also make alot more sense.)
I also ordered the items according to the JPwiki because they usually use the retail list instead of the LSB ordering by conquest/ID.)
Not gonna touch the formating otherwise.

Sources:
- https://www.bg-wiki.com/ffxi/Tavourine
- https://wiki-ffo-jp.translate.goog/html/9245.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US
- https://www.bg-wiki.com/ffxi/Version_Update_(07/11/2011)

zach2good committed:
[core] Replace relative includes

zach2good committed:
[cpp] Add contains_relative_include CI check

zach2good committed:
[ci] Add suppression for cppcheck:useInitializationList

zach2good committed:
[core] Add safety check to setExData

Skerxan committed:
[Shops] Audited Royal Armoury

Update

Skerxan committed:
[Shops] Audited Royal Armoury

Updated place 2 & 1 items.

TeoTwawki commented on sql/item_mods.sql:
```diff
@@ -2109,7 +2109,7 @@ INSERT INTO `item_mods` VALUES (10615,384,300); -- HASTE_GEAR: 300
 
 -- Rheic Schuhs
 INSERT INTO `item_mods` VALUES (10616,1,26);      -- DEF: 26
-INSERT INTO `item_mods` VALUES (10616,161,-3000); -- DMGPHYS: -3000
```
_eyes suspiciously and consults git-blame_

atom0s committed:
Fix custom menus not working with Japanese clients.
Add detection of event cancelled GM tell windows.
Add `is_event` parameter to custom menus 'onCancelled' callback function.

helixhamin committed:
Adds in many missing item mods.

helixhamin commented:
Sorry, think I messed up my commit. I did switch/checkout, under a new branch name, but I guess that it didn't push to the repository, and pushed all my new commits to the same PR, instead of a separate PR.
Sorry, not sure what to do next to resolve.

The new ones are from @vitus0123456 in https://github.com/LandSandBoat/server/discussions/3839. I had to correct some stuff, and added some stuff, but they did most of the work on the latest.

helixhamin commented:
Okay, I broke it...
No idea what to do now...

helixhamin commented:
Closing. I'll fix whatever my issues are and try again.
